doc/Media2.xml

```diff
@@ -2291,7 +2536,22 @@
           </listitem>
         </varlistentry>
       </variablelist>
-    </section>
+    <para>The following audio clip capabilites are available:</para>
+	<variablelist>
+        <varlistentry>
+          <term>MaxAudioClipLimit</term>
+          <listitem><para>Indicates the supported Maximum number of audio clips can be uploaded.</para></listitem>
+        </varlistentry>
+		<varlistentry>
+          <term>MaxAudioClipSize</term>
+          <listitem><para>Indicates Maximum size of the audio clip can be uploaded in KB.</para></listitem>
+        </varlistentry>
+		<varlistentry>
+          <term>SupportedAudioClipFormat</term>
+          <listitem><para>Indicates Supported audio clip formats by the device i.e WAV, MP3, OGG.</para></listitem>
```

Comment:
what is intended to show, decoders or also containers. OGG for example, is that running with vorbis or opus.

Reply:
the reason for SupportedAudioClipFormat is, client should know supported audio clip formats by the device, so that client can upload audio clips to the device what device supports. Yes, along with the container formats, we may need to provide respective decoding.

Reply:
SupportedAudioClipFormat  enumerations are defined which accommodates OGG-Opus, OGG-Vorbis, OGG-Flac, OGG-Speex along with WAV, MP3 containers.